COMMIT-GUIDELINES.md

```diff
@@ -0,0 +1,161 @@
+
+Breaking changes are denoted in the footer of the commit by starting it with `BREAKING CHANGE:` followed by a message about what the breaking change impacts.
+
+It is highly recommended to add a clear reason for the breaking change and what actions need to be taken when upgrading to the newer version. See the example under [Breaking change](#breaking-change) for an idea of how to communicate the change.
+
+It is important to note: Breaking changes can be added to any commit irregardless of the commit prefix, they are not just relegated to `fix` or `feat` commit prefixes as those aren't always relevant when introducing a breaking change.
+
+## Examples
+
+Good commit messages go along way in creating and maintaining a high quality project. They tell the story of how it grew and evolved overtime. They also help when investigating where things happened, should you ever have to go back in time. Commit messages are almost as important as high quality documentation and code comments.
+
+### Base structure
+
+The base structure of a commit message is:
+
+```
+prefix(optional-scope): subject
+
+body
+
+footer
+```
+
+### Commits with prefix and scope
+
+Adding a prefix and scope helps indicate the impact and theme of the commit and what it impacts.
+
+Commits can be simple:
+
+```
+chore(monorepo): add useWorkspaces to lerna.json
+```
+
+Commits can be more detailed:
+
+```
+fix(utils): isNumber function validation inverted
+
+This fix handles validating numbers are numbers and not numbers are `NaN`.
+
+- Add `undefined` and `null` checks to `isNumber` to guarantee they're
+  false.
+- Add unit tests to validate this case via the `date-validation-utils` function `isInvalidYear`.
+- Add unit tests to validate the `isNumber` util function.
+
+JIRA: DS-1234
+```
+
+Commits can have body text span multiple lines as paragraphs using new lines:
+
+```
+fix(utils): update retrieveEnumKeys function
+
+Updates to `isNumeric` invalided the logic this function was using,
+inverting it restores intended functionality.
+
+- Add unit tests to validate different types of enum cases as well as
+  `null` and `undefined`.
+```
+
+### Breaking change
+
+Adding `BREAKING CHANGE:` to the footer denotes the commit has a breaking change and that there should be a major version increment.
+
+```
+feat(ontario-card)!: added layout prop
+ - Updated unit tests
+ - Regenerated component definitions
+
+BREAKING CHANGE: `card-type` prop is now deprecated. Use `layout` prop instead.
+```
+
+### Commits with just a prefix
+
+Sometimes commits span more than a single scope but contain a number of changes within the same theme, or action, or the scope is ambiguous. When this happens it is recommended to break the commits down by scope, but in the case the commits impact multiple scopes the same commit message rules apply but the scope can be emitted.
```

Comment:
This is a good section but I think it can be written to flow a bit better. Let me knoww hat you think of my try at it below:

```
Sometimes commits can span more than a single scope but contain a number of changes within the same theme or action, or the scope is ambiguous. When this happens, it is recommended to break the commits down by scope. However, in cases where the commits impact multiple scopes, the scope can be omitted.  All other commit message rules still apply.
```

Also `emitted` should be `omitted`.